authentication-pre-request.js

```diff
@@ -0,0 +1,38 @@
+var { Header } = require('postman-collection')
+
+const oauth2Request = {
+    url: 'https://api.oregonstate.edu/oauth2/token',
+    method: 'POST',
+    header: 'Content-Type:application/x-www-form-urlencoded',
+    body:`grant_type=client_credentials&scope=
+      &client_id=${pm.collectionVariables.get('clientID')}
+      &client_secret=${pm.collectionVariables.get('clientSecret')}`
```

Comment:
Not important, but `pm.collectionVariables` are used many times in this script. Do you think we should desctrcture `collectionVariables` at the very top?